contracts/implementations/streamr/StreamrSacdListener.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+
+import "../../interfaces/ISacdListener.sol";
+import "./VehicleStream.sol";
+
+contract StreamrSacdListener is ISacdListener {
+
+    uint256 private constant VEHICLE_SUBSCRIBE_LIVE_DATA_PERMISSION_BITMASK = 1 << 6;
```

Comment:
Unfortunately we use two bits for each permission. Getting the stream permission means your mask has

```
3 << 12
```

in it. @LorranSutter double check me.